presto-spark-base/src/test/java/com/facebook/presto/spark/execution/TestNativeExecutionProcess.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution;
+
+import com.facebook.airlift.json.JsonCodec;
+import com.facebook.presto.Session;
+import com.facebook.presto.client.ServerInfo;
+import com.facebook.presto.execution.TaskId;
+import com.facebook.presto.execution.TaskManagerConfig;
+import com.facebook.presto.spark.execution.http.TestPrestoSparkHttpClient;
+import com.facebook.presto.spark.execution.property.NativeExecutionConnectorConfig;
+import com.facebook.presto.spark.execution.property.NativeExecutionNodeConfig;
+import com.facebook.presto.spark.execution.property.NativeExecutionSystemConfig;
+import com.facebook.presto.spark.execution.property.PrestoSparkWorkerProperty;
+import org.testng.annotations.Test;
+
+import java.net.URI;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static com.facebook.airlift.http.client.HttpUriBuilder.uriBuilder;
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static java.util.concurrent.Executors.newScheduledThreadPool;
+import static java.util.concurrent.Executors.newSingleThreadExecutor;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotSame;
+
+public class TestNativeExecutionProcess
+{
+    private static final JsonCodec<ServerInfo> SERVER_INFO_JSON_CODEC = JsonCodec.jsonCodec(ServerInfo.class);
+    private static final URI BASE_URI = uriBuilder()
+            .scheme("http")
+            .host("localhost")
+            .port(8080)
+            .build();
+
+    @Test
+    public void testNativeProcessRelaunch()
```

Comment:
could we change the test name to something relevant (e.g `testNativeProcessIsAlive`)?